[2020-06-17 18:25:53,881] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:08:00+00:00 [queued]>
[2020-06-17 18:25:53,941] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:08:00+00:00 [queued]>
[2020-06-17 18:25:53,941] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 18:25:53,941] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-17 18:25:53,941] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 18:25:54,127] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): generate_file> on 2020-06-17T03:08:00+00:00
[2020-06-17 18:25:54,129] {standard_task_runner.py:53} INFO - Started process 2353 to run task
[2020-06-17 18:25:54,428] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: generator_v0.generate_file 2020-06-17T03:08:00+00:00 [running]> strx
[2020-06-17 18:25:54,472] {logging_mixin.py:112} INFO - Generating  1592432754 ...
[2020-06-17 18:25:54,472] {logging_mixin.py:112} INFO - Finished.
[2020-06-17 18:25:56,474] {python_operator.py:114} INFO - Done. Returned value was: _
[2020-06-17 18:25:56,639] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=generator_v0, task_id=generate_file, execution_date=20200617T030800, start_date=20200617T222553, end_date=20200617T222556
[2020-06-17 18:26:03,684] {logging_mixin.py:112} INFO - [2020-06-17 18:26:03,683] {local_task_job.py:103} INFO - Task exited with return code 0
